lib/common/exceptions/internal/evss_forbidden.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Common
```

Comment:
I like the name change for this PR and will implement.  This gets me thinking that the architecture should be the other way around: base exceptions should always be logged and there should be some sub-folder (or maybe a Mixin?) like "non_loggable" and the `SKIP_SENTRY_EXCEPTION_TYPES` logic would exclude all of those.